### ISSUE TITLE
Converting SDA to Steam Mobile App

### ISSUE THREAD

**Issue by Newcool1230** [Jessecar96/SteamDesktopAuthenticator#1084]
Is there a straightforward tutorial to convert SDA to Steam Mobile App without having the 7 day trade cooldown?

Is there a way to input maFiles into the mobile app? or is there anyway to continue using steam guard on desktop?

**Comment by dyc3** [comment]
You can "transfer" the authenticator to the mobile app with just a 2 day trade ban. Do not remove the authenticator and re add it, that gets you the longer trade ban.

The mobile app walks you through it. Just log in and try to add an authenticator and follow the prompts.

**Comment by spookykokojunge** [comment]
So is SDA now unsuported lik will i still be able to use it or what exactly happend ?

**Comment by dyc3** [comment]
Nope. If you transfer the authenticator, the codes from your old authenticator in SDA are no longer valid.

If you are interested, I maintain steamguard-cli. It's capable of reading the exactly the same maFiles that SDA does, so you don't have to deal with a trade ban. It doesn't have a UI, so if you aren't really comfortable using a terminal, I would recommend you use the steam mobile app. You can download it here: https://github.com/dyc3/steamguard-cli/releases

Read the quickstart here: https://github.com/dyc3/steamguard-cli/blob/master/docs/quickstart.md

**Comment by dyc3** [comment]
Steam's 2fa code algorithm is non standard, so if you bring it in to most applications they won't generate the right code.

**Comment by C-O-S** [comment]
Just for reference:  

I've also used [KeePass](https://keepass.info/) with the [KeePassOTP](https://github.com/Rookiestyle/KeePassOTP) plugin successfully.

[Instructions](https://github.com/Rookiestyle/KeePassOTP/wiki/Using-KeePassOTP-with-Steam)

The only real issue is you can't acknowledge confirmations for trades, etc. using this route - but it does give you the proper 2FA codes needed to logon to Steam, and stores everything within an encrypted database.

**Comment by spdfnet** [comment]
Or use KeepassXC directly.

**Comment by C-O-S** [comment]
> Or use KeepassXC directly.

True enough! :  From the [User Guide](https://keepassxc.org/docs/KeePassXC_UserGuide#_adding_totp_to_an_entry)

**Comment by NUCL3ARN30N** [comment]
Another alternative which supports trade confirmations is https://github.com/JustArchiNET/ArchiSteamFarm.
It is well maintained.

**Comment by freefrank** [comment]
maFiles can't be imported into Steam's own app, but they work as-is in a third-party one. I wrote AVA for this exact case — import the maFile, no re-link, so no 7-day cooldown. Confirmations and trade offers are in there.

Android, Windows, Linux, macOS: https://github.com/freefrank/AnotherVaporAuth

I'm the author, so weigh accordingly.
